README.md

```diff
@@ -82,6 +83,20 @@ To See mail that has been sent on the staging server you must ssh tunnel into th
 
 Once the tunnel is open you can see the mail that has been sent on [staging1 here](http://localhost:1082/) and on [staging 2 here](http://localhost:1083/)
 
+### Mail on Production
+Emails on production are sent via [Pony Express](https://github.com/pulibrary/pul-it-handbook/blob/f54dfdc7ada1ff993a721f6edb4aa1707bb3a3a5/services/smtp-mail-server.md).
+
+### Mail on Development
+Mailcatcher is a gem that can also be installed locally.  See the [mailcatcher documentation](https://mailcatcher.me/) for how to run it on your machine.
+
+Once mailcatcher is running set your config/environments/development.rb to include:
+```
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: "localhost",
+    port: 1025
+  }
+```
```

Comment:
Is there an argument against just making mailcatcher part of the default dev environment? I know that once I've made changes to my local config, unless the file is gitignored, I'm probably going to check it in at some point by mistake. It's hard to break the `git add .` habit.

Reply:
This was discussed during the check-in meeting, and might it be possible for me to please issue a pull request setting this as the default for the application instead? I apologize, I can issue the PR proposing these changes momentarily.

Reply:
Please see https://github.com/pulibrary/pdc_describe/pull/872